### ISSUE TITLE
REpeat

### ISSUE THREAD

**Issue by handicraftsman** [VictiniX888/Illegal-Mod-Sites#50]
xn--2-8sbausglk2acux xn--p1ai = 2майнкрафтер.рф. Also i see GIANT lags at this site (ads, of course)

**Comment by VictiniX888** [comment]
Russian sites have weird URLs :P 
Both work, however if I type the latter, it still gets converted to xn--2-8sbausglk2acux xn--p1ai
I'll never get how Russian works

**Comment by Sollace** [comment]
@VictiniX888 
Seems like your browser doesn't understand Russian characters. When I go to the first one it gets converted to the second one.

**Comment by bilde2910** [comment]
I was the one who added this site to the list a while ago. Only one of the variations would technically be required, as both are just different variations of the same name, however at the time, there were a few reasons which warranted adding both of the variations to the list. These two stood out as the main reasons:

* Different rendering between browsers: Not all the browsers render Unicode domain names the same way. At the time of adding the site (and I don't think it has changed since), Chrome would display the Punycode variation (xn--...), and Firefox would display the Unicode/Cyrillic variation (2ма...) in the address bar. When people look up sites in the list, they would look for the variation displayed in their address bar. Most people don't even know that domains can be represented in different ways, and if you asked the average Minecraft user if there is a difference between the two domains, they would think there was, as the concept of Punycode is unknown to many. Both variations were added so people from all browsers would be able to look up the site in the list easily.
* Different internal representation in the browsers: When I was updating the list of sites in the Stop Mod Reposts browser extensions to include this domain, I noticed, to much surprise, that Chrome and Firefox also handle the domain names differently in the internal browser code. Adding only the Punycode variation to the site list made Chrome, but not Firefox display the alert on the site, and replacing it with the Unicode variation made it the other way around. In order for both of the browsers to recognize the site as illegal and throw up a warning, both of the domain representations had to be added to the lists. As the JSON representation of the site list used in the extensions is generated directly from the GitHub list using regex, having both variations on the list makes my job as maintainer of the extensions that much easier in that I don't have to find and add the other variation of the domain name to the list manually every time I want to update the extension.

The reason Chrome even displays Punycode at all is to prevent [IDN homograph attacks](https://en.wikipedia.org/wiki/IDN_homograph_attack). Some letters look really similar or identical across multiple alphabets, e.g. the letter а (from Cyrillic) and a (from Latin). It's impossible to see a difference. This can result in situations where one domain name looks identical to another, yet one is owned by a phishing group and the other is a legitimate business. Forcing Punycode rendering makes this a lot more apparent (see exаmple.com vs xn--exmple-4nf.com).
